code-challenges.test.js

```diff
@@ -20,11 +32,61 @@ const fibLength1 = 6
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("greaterFib", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(greaterFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(greaterFib(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+// -> ReferenceError: greaterFib is not defined
+
 // b) Create the function that makes the test pass.
 
+/*
+Create a function that takes in one parameter as an argument
+The argument that the parameter is taking in is a number data type that is greater than 2
+Find and use a method that takes in the fibonacci sequence
+Use the number data type as the length of how many numbers in the fibonacci sequence will be used
+The Array.from() method is a static method that creates a new array. 
+Return the numbers from the fibonacci sequence inside of an array
+*/
+
+const greaterFib = (number) => {
+  let fibNumber = (number - 1) + (number - 2)
+  return Array.from(fibNumber)
+}
+  
+  /*
+
+      - Expected  - 8
+    + Received  + 1
+
+    - Array [
+    -   1,
+    -   1,
+    -   2,
+    -   3,
+    -   5,
+    -   8,
+    - ]
+    + Array []
+
+    Test Suites: 1 failed, 1 total
+Tests:       1 failed, 2 passed, 3 total
+  
+    I am currently stuck at this point. I understand that we need the equation for the fibonacci sequence, but if I take out the Array.from() method, it will only return the number in the sequence. But if I tried inputting predetermined values to start the sequence or using an if statement to confirm that the number argument is larger than two, I either get more numbers than expected or multiple NaN's. 
```

Comment:
Array.from is going to take in something and break it up into an array.  In this case, we are taking in a number that will be the length of the array.  One of the most straight forward approaches is using a for loop for this problem.